bgpd/bgp_aspath.c

```diff
@@ -1558,6 +1561,38 @@ struct aspath *aspath_prepend(struct aspath *as1, struct aspath *as2)
 	/* Not reached */
 }
 
+/* insert aspath exclude in head of orphan exclude list*/
+void as_exclude_set_orphan(struct aspath_exclude *ase)
+{
+	ase->exclude_aspath_acl = NULL;
+	as_list_list_add_head(&as_exclude_list_orphan, ase);
+}
+
+void as_exclude_orphan_remove(struct aspath_exclude *ase)
```

Comment:
NB: just a cosmetic change to be `as_exclude_remove_orphan()`, same as `as_exclude_set_orphan()`. It's a bit annoying when the operation keyword is at the random place of the function's name.

Reply:
taken into account